portal/component-tests/test-helpers/with-contact-us-email-address.component-tests.js

```diff
@@ -0,0 +1,48 @@
+const pageRenderer = require('../pageRenderer');
+
+const page = 'login/temporarily-suspended.njk';
```

Comment:
Might be misunderstanding but shouldn't page be passed into `withContactUsEmailAddressTests`